spec/index.bs

```diff
@@ -1894,6 +1900,111 @@ dictionary GPUVertexAttributeDescriptor {
 };
 </script>
 
+## {{GPUPipelineBase/getBindGroupLayout()|getBindGroupLayout}} ## {#get-bind-group-layout}
+
+<dl dfn-type="method" dfn-for="GPUPipelineBase">
+    : <dfn>getBindGroupLayout(unsigned long index)</dfn>
+    ::
+        <div algorithm="GPUProgrammablePipeline.getBindGroupLayout(unsigned long index)">
+            1. If |index| is greater or equal to {{GPULimits/maxBindGroups}} :
+
+                1. Throw a {{RangeError}}.
+
+            1. If |this|.getBindGroupLayout has already been called with |index|:
+
+                1. Return the same object.
+
+            1. If |this| is an error:
+
+                1. Return an error {{GPUBindGroupLayout}}.
+
+            1. Let |bindings| be an empty sequence.
+            1. For each {{GPUProgrammableStageDescriptor}} |stage| in the descriptor for |this|:
```

Comment:
`in the descriptor` - what descriptor is this?

Reply:
The `GPURenderPipelineDescriptor` or `GPUComputePipelineDescriptor`. I wasn't sure how to phrase this precisely. We could maybe have an internal slot in pipeline to store the list of (entryPoint + shaderModule + specialization constants).